### DISCUSSION TITLE
Support for Tuya Video Doorbells

### DISCUSSION THREAD

**Discussion by JasperMC** [ct-Open-Source/tuya-convert#859] [Ideas]
Hello there!

I was wondering if Tuya Convert is planning to support Tuya Video Doorbells as well. These are readily available from AliExpress, Amazon and now also in some retail stores, e.g. https://www.action.com/nl-nl/p/dummy-3003046/

Most of these devices are locked to the Tuya platform and don’t support Google Home or others.

Unfortunately, I’m unaware if these also run ESP8266’s or something else entirely.

Willing to help with testing :)

**Comment by kueblc** [comment]
These do not use the ESP82xx, they tend to run something a bit heftier to provide a real time video stream
